### PR TITLE
feat(showcase): design-factory integration

### PR DESCRIPTION
**Pull request by fpaul-1A**
## Proposed change

Integration of design-factory for the showcase app
~~We need a fix in `@design-factory/design-factory` package.json to properly export the sass file~~
https://fpaul-1a.github.io/otter

## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

File: apps/showcase/package.json [hunk 1]
```diff
@@ -75,6 +78,7 @@
     "@angular/cli": "~16.2.0",
     "@angular/compiler-cli": "~16.2.0",
     "@ng-bootstrap/ng-bootstrap": "^15.1.1",
+    "@ng-select/ng-select": "~11.0.0",
```
[the remark this answers is not in the document]
Comment (fpaul-1A) [reply apps/showcase/package.json:81]: it was added by the `ng-add` of design-factory, and yes it's a non-optional peerDep

File: package.json [hunk 2]
```diff
@@ -158,6 +158,7 @@
     "@commitlint/cli": "^17.0.0",
     "@commitlint/config-conventional": "^17.0.0",
     "@compodoc/compodoc": "^1.1.19",
+    "@design-factory/design-factory": "^15.2.2",
```
[the remark this answers is not in the document]
Comment (fpaul-1A) [reply package.json:161]: it was required to run the `yarn unplug`

File: apps/showcase/package.json [hunk 3]
```diff
@@ -51,7 +52,9 @@
     "@o3r/rules-engine": "workspace:^",
     "@o3r/testing": "workspace:^",
     "@popperjs/core": "^2.11.5",
-    "bootstrap": "^5.3.1",
+    "ag-grid-angular": "~29.3.5",
+    "ag-grid-community": "~29.3.5",
+    "bootstrap": "5.2.0",
```
[the remark this answers is not in the document]
Comment (fpaul-1A) [reply apps/showcase/package.json:57]: the peerDependencies on `@design-factory` is `bootstrap: 5.2.0`, I can fix it in the yarnrc, but should we ?

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (fpaul-1A) [reply package.json:163]: actually the sass-loader only finds the deps if I put them at root level, the unplug is not needed

[the remark this answers is not in the document]
Comment (kpanot) [reply package.json:163]: This really strange :S

[the remark this answers is not in the document]
Comment (fpaul-1A) [reply package.json:163]: create issue https://github.com/AmadeusITGroup/otter/issues/979 to continue the investigation

**Review by cpaulve-1A — APPROVED** [review]

**Review by matthieu-crouzet — APPROVED** [review]

**Pull request merged**